Add feedback popup implementation.

mprasanjith opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description

<!--- Describe your changes in detail -->

Implements a basic popup which links to a Google form and Github to simplify how we collect feedback.

## Motivation and Context

<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->

#281 

## How Has This Been Tested?

<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

Tested locally using start-xdai.

## Screenshots (if appropriate):

![Screenshot 2021-06-30 at 21-57-36 Fixed Price Sale Token – FPST Mesa](https://user-images.githubusercontent.com/25136207/123997706-4baced00-d9ee-11eb-967f-310075c6bc06.png)

## Types of changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

rossneilson reviewed (approved):
This looks fantastic, good job!

One small thing would be on mobile. Currently it clashes with the action buttons at the bottom (Also looks like there is another bug on mobile, it is showing withdraw/claim instead of purchase). Not too important but a quick solution would just be to add extra margin on mobile. In future it may be better to add it to the burger menu for mobile. 

Also in future an animation might be nice on open and close, but low priotity.

mprasanjith commented:
I updated the mobile design a bit, and the trigger button will now show only when the burger menu is open. Also added a close button to the popup. Animations will definitely be a nice addition down the line.

martinkrung commented:
Nice and Clean. I like it!
